zkclient/help.go

```diff
@@ -17,10 +17,8 @@ var (
 	cmdAcceptnewcert = leader + "acceptnewcert"
 	cmdVersion       = leader + "version"
 	cmdHelp          = leader + "help"
-	cmdFetch         = leader + "fetch"
```

Comment:
We need to keep /fetch and /kx for closed systems that don't provide directory services.

Reply:
Yes, with the recently introduced directory toggle we need to keep /kx and /fetch. I didn't expect a zkserver toggle when working on this code months ago. I will basically have to redo autokx.